test/support/fixtures/chainweb/local.json

```diff
@@ -1,24 +1,24 @@
 {
-  "continuation": null,
-  "gas": 7,
+  "gas": 6,
+  "result": {
+      "status": "success",
+      "data": 11
+  },
+  "reqKey": "SvVzKL5KC0r1JdIkJDhMHT8vZhmO2CdTxPKX-6YmGG0",
   "logs": "wsATyGqckuIvlm89hhd2j4t6RMkCrcwJe_oeCYr7Th8",
   "metaData": {
-    "blockHeight": 2815727,
-    "blockTime": 1671054330981668,
-    "prevBlockHash": "asisNr3nuU0t357i2bxMVUiIWDVHAMtncJHtmyENbio",
-    "publicMeta": {
-      "chainId": "0",
-      "creationTime": 1667249173,
-      "gasLimit": 1000,
-      "gasPrice": 1.0e-6,
-      "sender": "k:554754f48b16df24b552f6832dda090642ed9658559fef9f3ee1bb4637ea7c94",
-      "ttl": 28800
-    }
-  },
-  "reqKey": "-1npoTU2Mi71pKE_oteJiJuHuXTXxoObJm8zzVRK2pk",
-  "result": {
-    "data": 11,
-    "status": "success"
+      "publicMeta": {
+          "creationTime": 1671462208,
+          "ttl": 28800,
+          "gasLimit": 1000,
+          "chainId": "1",
+          "gasPrice": 1.0e-6,
+          "sender": "k:d1a361d721cf81dbc21f676e6897f7e7a336671c0d5d25f87c10933cac6d8cf7"
+      },
+      "blockTime": 1671546034831940,
+      "prevBlockHash": "Y6Wj0sxJpdV8M-3ihAbzUka57Wv5ZV2Uez6H_6_WeeY",
+      "blockHeight": 3303861
   },
+  "continuation": null,
   "txId": null
-}
+}
```

Comment:
@EdwinGuayacan please add a blank line at the end of the file. You can configure your editor to make this for you!